.ibm/pipelines/resources/config_map/configmap-app-config-rhdh.yaml

```diff
@@ -10,17 +10,6 @@ data:
         iconLogo: "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBzdGFuZGFsb25lPSJubyI/Pgo8IURPQ1RZUEUgc3ZnIFBVQkxJQyAiLS8vVzNDLy9EVEQgU1ZHIDIwMDEwOTA0Ly9FTiIgImh0dHA6Ly93d3cudzMub3JnL1RSLzIwMDEvUkVDLVNWRy0yMDAxMDkwNC9EVEQvc3ZnMTAuZHRkIj4KPCEtLSBDcmVhdGVkIHVzaW5nIEtyaXRhOiBodHRwczovL2tyaXRhLm9yZyAtLT4KPHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIAogICAgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiCiAgICB4bWxuczprcml0YT0iaHR0cDovL2tyaXRhLm9yZy9uYW1lc3BhY2VzL3N2Zy9rcml0YSIKICAgIHhtbG5zOnNvZGlwb2RpPSJodHRwOi8vc29kaXBvZGkuc291cmNlZm9yZ2UubmV0L0RURC9zb2RpcG9kaS0wLmR0ZCIKICAgIHdpZHRoPSI4MHB0IgogICAgaGVpZ2h0PSI4MHB0IgogICAgdmlld0JveD0iMCAwIDgwIDgwIj4KPGRlZnMvPgo8dGV4dCBpZD0ic2hhcGUwIiBrcml0YTp1c2VSaWNoVGV4dD0idHJ1ZSIgdGV4dC1yZW5kZXJpbmc9ImF1dG8iIGtyaXRhOnRleHRWZXJzaW9uPSIzIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxLjU5OTk5OTk5OTk5OTk5LCA2Mi44MTI1KSIgZmlsbD0iI2ZmZmZmZiIgc3Ryb2tlLW9wYWNpdHk9IjAiIHN0cm9rZT0iIzAwMDAwMCIgc3Ryb2tlLXdpZHRoPSIwIiBzdHJva2UtbGluZWNhcD0ic3F1YXJlIiBzdHJva2UtbGluZWpvaW49ImJldmVsIiBsZXR0ZXItc3BhY2luZz0iMCIgd29yZC1zcGFjaW5nPSIwIiBzdHlsZT0idGV4dC1hbGlnbjogc3RhcnQ7dGV4dC1hbGlnbi1sYXN0OiBhdXRvO2ZvbnQtZmFtaWx5OiBSZWQgSGF0IE1vbm87Zm9udC1zaXplOiA2NDtmb250LXdlaWdodDogNzAwOyI+PHRzcGFuIHg9IjAiPlFFPC90c3Bhbj48L3RleHQ+Cjwvc3ZnPgo="
-        theme:
```

Comment:
IMHO, it would be good to have both types of tests: using the default configuration and loading a specific custom theme. By removing those lines, we lose the guarantee that configuring a custom theme in place is working properly.
@josephca wdyt?

Reply:
Yeah, that is why I made that call out in the description. I am fine with what ever approach we want to do. Let me also double check to see if defining a custom theme dynamic plugin and configuring the theme in the app config will merge or overwrite. If they allow for merging, then I might be able to update this PR to check for both

Reply:
@gustavolira It looks like I can make an update to the dynamic plugin config to use both the dynamic plugin theme as well as the app-config theme. 
![Screenshot from 2024-10-24 09-44-21](https://github.com/user-attachments/assets/0912d093-7c76-4f7f-9b22-123e39ed0c22)
Let me update the tests to cover both scenarios now.